UnityPy/helpers/Tpk.py

```diff
@@ -4,22 +4,23 @@
 from importlib.resources import open_binary
 from io import BytesIO
 from struct import Struct
-from typing import Any, Dict, List, Tuple
+from typing import Any, Dict, List, Optional, Tuple
 
 from .TypeTreeHelper import TypeTreeNode
 
-TPKTYPETREE: TpkTypeTreeBlob = None
-CLASSES_CACHE: Dict[Tuple[int, tuple], TypeTreeNode] = {}
-NODES_CACHE: Dict[TpkUnityClass, TypeTreeNode] = {}
-
 
-def init():
+def init_tpk_typetree():
     with open_binary("UnityPy.resources", "uncompressed.tpk") as f:
         data = f.read()
-
-    global TPKTYPETREE
     with BytesIO(data) as stream:
-        TPKTYPETREE = TpkFile(stream).GetDataBlob()
+        blob = TpkFile(stream).GetDataBlob()
+        assert isinstance(blob, TpkTypeTreeBlob)
+        return blob
+
+
+TPKTYPETREE: TpkTypeTreeBlob = init_tpk_typetree()
```

Comment:
This doesn't work as TpkFile isn't defined yet on function call.
It simply errors.

```
Exception has occurred: NameError
name 'TpkFile' is not defined
```